.coderabbit.yaml

```
@@ -0,0 +1,191 @@
+  suggested_reviewers: false
+  auto_assign_reviewers: false
+  poem: false
+
+  auto_title_placeholder: "@coderabbitai"
+  review_status: true
+  commit_status: true
+  fail_commit_status: false
+  collapse_walkthrough: false
+  changed_files_summary: true
+
+  labeling_instructions: [ ]
```

YAMLlint (1.37.1)

[error] too many spaces inside empty brackets

(brackets)